docs/usage_and_configuration/the_basics.md

```diff
@@ -200,6 +200,37 @@ project.
 "No". _Black_ is all about sensible defaults. Applying those defaults will have your
 code in compliance with many other _Black_ formatted projects.
 
+````{note}
+If you are using the linked configuration feature i.e. specifying `config` in the black config,
+it would be overwritten by the parent config i.e. the config which contains the `config` specifying
+the linked configuration.
+
+Say `black.config` is the parent config containing
+```{code-block} toml
+---
+lineno-start: 1
+emphasize-lines: 2,3
+---
+[tools.black]
+config = "linked_config.toml"
+color = true
+```
+
+and the linked configuration containing
+```{code-block} toml
+---
+lineno-start: 1
+emphasize-lines: 2
+---
+[tools.black]
+color = false
+line-length = 88
+target-version = ['py36', 'py37', 'py38']
+```
+
+The resulting value for `color` would be `true` as specified in the parent config.
```

Comment:
Surely the "parent" configuration is the linked configuration here, which the child inherits and overrides. But now the behavior is starting to make sense 👍

Reply:
I think we should avoid talking about the "parent" config because different people understand it to be either the first or the second config.